### ISSUE TITLE
Can not evaluate and save result on test set

### ISSUE THREAD

**Issue by itruonghai** [PRBonn/contrastive_association#2]
Can you explain what is coordinate_T in this [line](https://github.com/PRBonn/contrastive_association/blob/649693494197c8d3948252daee6767b66a89c868/cont_assoc/models/ps4d_models.py#L66)

**Comment by rmarcuzzi** [comment]
Hi! 
thanks for checking out our code! That variable was present in the original code and I forgot to remove it here! It was supposed to be the translated coordinates to the global frame. They are not used in the end.
I hope this answers your question, I've modified the code, thank you!

**Comment by itruonghai** [comment]
@rmarcuzzi Will you release the training code for the instance segmentation part?

**Comment by rmarcuzzi** [comment]
Hi!
I didn't train the panoptic segmentation network since in my approach I only use it frozen. I took the pretrained weights from the original implementation and ported them into mine, you can download them [here](https://www.ipb.uni-bonn.de/html/projects/contrastive_instance_association/panoptic_pq_564.pth).
However, it should be similar to the original repo using the same exact loss function.
